inclusive-design/idrc
# feat: add Connect page (resolves #80)

## Description

Adds "Connect" page.

## Steps to test

1. Visit https://deploy-preview-89--idrc.netlify.app/connect.

**Expected behavior:** Layout matches designs.

## Additional information

Not applicable.

## Related issues

Resolves #80.
Depends on #88.
